From 224cdc61e73fb81aa67f1377192de9041ce68f1a Mon Sep 17 00:00:00 2001
From: jmulliga <james.mulligan@yale.edu>
Date: Thu, 2 Mar 2017 14:43:57 -0500
Subject: [PATCH] print out warning in tender that new correction framework is
 preferred

---
 TENDER/TenderSupplies/AliEMCALTenderSupply.cxx | 16 ++++++++++++----
 1 file changed, 12 insertions(+), 4 deletions(-)

diff --git a/TENDER/TenderSupplies/AliEMCALTenderSupply.cxx b/TENDER/TenderSupplies/AliEMCALTenderSupply.cxx
index 0412e4c..57f3b59 100644
--- a/TENDER/TenderSupplies/AliEMCALTenderSupply.cxx
+++ b/TENDER/TenderSupplies/AliEMCALTenderSupply.cxx
@@ -304,9 +304,17 @@ Bool_t AliEMCALTenderSupply::RunChanged() const
 void AliEMCALTenderSupply::Init()
 {
   // Initialise EMCAL tender.
-
-  if (fDebugLevel>0) 
-    AliWarning("Init EMCAL Tender supply"); 
+  
+  if (fDebugLevel>0) {
+    AliWarning("Init EMCAL Tender supply");
+    AliWarning("======================================================================================");
+    AliWarning("--------------------------------------------------------------------------------------");
+    AliWarning("============  The EMCal Tender is no longer supported for development!  ==============");
+    AliWarning("=========  It is recommended to use the new EMCal Correction Framework!  =============");
+    AliWarning("===  See http://alidoc.cern.ch/AliPhysics/master/_r_e_a_d_m_eemc_corrections.html  ===");
+    AliWarning("--------------------------------------------------------------------------------------");
+    AliWarning("======================================================================================");
+  }
   
   if (fConfigName.Length()>0 && gROOT->LoadMacro(fConfigName) >=0) {
     AliDebug(1, Form("Loading settings from macro %s", fConfigName.Data()));
@@ -355,7 +363,7 @@ void AliEMCALTenderSupply::Init()
   }
   
   if (fDebugLevel>0){
-    AliInfo("Emcal Tender settings: ======================================"); 
+    AliInfo("Emcal Tender settings:");
     AliInfo("------------ Switches --------------------------"); 
     AliInfo(Form("BadCellRemove : %d", fBadCellRemove)); 
     AliInfo(Form("ExoticCellRemove : %d", fRejectExoticCells)); 
-- 
2.10.1 (Apple Git-78)

